perf_test

```diff
@@ -69,11 +69,11 @@ def add_peak(results, peaks, bs, key):
                                'CPU System' : peak[f'CPU System'],
                               })
 
-def pretty_print(results, title):
+def pretty_print(results, title, args):
```

Comment:
Passing args from argparse to a function is an anti-pattern.  In fact it doesn't, look like this generic table printing function should know anything about the data in the table.

Maybe just do something like:

```python
subtitle = f"(disks:{args.disks}, level:{args.level})"
...
    pretty_print(results['BW']['write'], f'Seq Write BW {subtitle}')
```